client/my-sites/media/main.jsx

```diff
@@ -358,6 +360,19 @@ class Media extends Component {
 				{ mediaId && site && site.ID && <QueryMedia siteId={ site.ID } mediaId={ mediaId } /> }
 				<PageViewTracker path={ this.getAnalyticsPath() } title="Media" />
 				<DocumentHead title={ translate( 'Media' ) } />
+				{ this.props.selectedSite.is_private && this.props.selectedSite.is_wpcom_atomic && (
+					<Notice
+						showDismiss={ false }
+						status="is-info"
+						text={ translate(
+							'Your site is Private and the image CDN is disabled. If image thumbnails do not display, switch to Coming Soon mode.'
+						) }
+					>
+						<NoticeAction href="/support/settings/privacy-settings/#private" external>
```

Comment:
Should we or can we use `<InlineSupportLink>` here? It might be a nicer effect to load this inline.

Reply:
Ohh, great suggestion; That one would definitely be better for UX. I'll work on implementing it!

Reply:
Added the `InlineSupportLink` component with the existing `privacy` link as `supportContext`. Thanks for the suggestion it looks much better! 🙌